### PR TITLE
Replace NIB links with docs/previous equivalent

### PR DESCRIPTION
**Pull request by nemrism**
## Summary

Replaced several MSDN NIB links with their https://docs.microsoft.com/previous-versions/ equivalent.

Contributes to dotnet/docs#1787 & dotnet/docs#2037

### CODE REVIEW

File: xml/System.Web.Configuration/HttpModuleAction.xml [hunk 1]
```diff
@@ -31,7 +31,7 @@
   
  The following configuration file example shows how to specify values declaratively for the `httpModules` section.  
   
- Refer to the [NIB: How to: Create and Configure an HTTP Module](https://msdn.microsoft.com/library/4174fd13-a9c4-4ff6-baa5-3f7d7fc73ef3) to see how to build and configure an HTTP module.  
+ Refer to the [How to: Create and Configure an HTTP Module](/previous-versions/dotnet/netframework-3.0/tfd6k449(v=vs.85)) to see how to build and configure an HTTP module.
```
Comment (mairaw) [inline xml/System.Web.Configuration/HttpModuleAction.xml]: You can use the full URL for these. They wouldn't work in the offline books.

```suggestion
 Refer to the [How to: Create and Configure an HTTP Module](https://docs.microsoft.com/previous-versions/dotnet/netframework-3.0/tfd6k449(v=vs.85)) to see how to build and configure an HTTP module.  
```

Comment (nemrism) [reply xml/System.Web.Configuration/HttpModuleAction.xml]: Ok I'll change the others also.
Does this also apply to the [dotnet/docs](https://github.com/dotnet/docs) repository?

Comment (mairaw) [reply xml/System.Web.Configuration/HttpModuleAction.xml]: for previous-versions links, yes

Comment (nemrism) [reply xml/System.Web.Configuration/HttpModuleAction.xml]: I think I've forgotten this rule recently... I may have to check one or two recent commits on the docs repo!

Comment (mairaw) [reply xml/System.Web.Configuration/HttpModuleAction.xml]: It's not a big deal. Don't worry!

**Comment by nemrism** [comment]
Hi there,
any news about this PR validation?

**Pull request merged**